src/sink/kinesis.rs

```diff
@@ -111,7 +111,7 @@ impl Sink<KinesisConfig> for Kinesis {
     ///
     /// If the given record would put the buffer at capacity, then the contents
     /// are first flushed before the given record is added.
-    fn deliver_raw(&mut self, _encoding: metric::Encoding, bytes: Vec<u8>) {
+    fn deliver_raw(&mut self, order_by: u64, _encoding: metric::Encoding, bytes: Vec<u8>) {
```

Comment:
Hmm, at this point, why not have deliver_raw take an actual Raw, pull out what's needed in the function body rather than pass arguments?

Reply:
I had thought about it.

I was just really looking forward to a straight forward PR that didn't require me to make API changes.  :D

Reply:
100% fair.